README.md

```diff
@@ -446,18 +435,18 @@ ucan-selector = "*" / cid
 
 `ucan:*` represents all of the UCANs in the current proofs array. If selecting a particular proof (i.e. not the wildcard), then the [CID](#65-content-identifiers) MUST be used. In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs (`prf`) field.
 
-## 4.2 `owned`
+## 4.2 `own`
 
-The `owned` URI subscheme defines how to address all resources of a particular URI scheme owned by a particular [DID subject](https://www.w3.org/TR/did-core/#dfn-did-subjects). The DID subject MUST NOT contain any other characters, such as query parameters or fragments. The `scheme-selector` MUST either be the wildcard (`*`) selector or a URI scheme (such as `dns`, `mailto`, and `telnet`). 
+The `own` URI subscheme defines how to address all resources of a particular URI scheme owned by a particular [DID subject](https://www.w3.org/TR/did-core/#dfn-did-subjects). The DID subject MUST NOT contain any other characters, such as query parameters or fragments. The `scheme-selector` MUST either be the wildcard (`*`) selector or a URI scheme (such as `dns`, `mailto`, and `telnet`). 
 
 ``` abnf
-owned = "owned://" <did-subject> "/" scheme-scope
+own = "own://" <did-subject> "/" scheme-scope
 scheme-selector = "*" / <scheme>
 ```
 
 ```
-owned://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/*
-owned://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/dns
+own://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/*
+own://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV/dns
```

Comment:
We could potentially pull this out into its own spec, as it's not a hard requirement for the core spec.

Reply:
@expede This is kind of a thing that we probably would have expressed as follows instead

1. Can do anything `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV` could do
```
{ can: "*", with: "did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV" }
```

2. Can do same DNS stuff as `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV`

```
{ can: "*", with: "dns://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV" }
```


If expressed as above `own:` becomes unnecessary, for the same reason as `my` is unnecessary - if issuer is same ID it can self issue, otherwise it must be delegated from the did.

Reply:
@Gozala I see what you're trying to do, but both break the URI schemes which are very clearly defined in their own specs. For example, `dns://did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV` would need to resolve `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV` as a DNS record, which it's not formatted as, and so even a DNS parser would fail. What `own` gives you here is a signal that we're not using the `dns:` URI scheme.

Reply:
FWIW, Fission has done variants of both of the above in the past, but I've come to realize that once we switched to URI resources, we should probably honour the URIs that we say they are, otherwise there's lots of sharp edges, potential for confusion, and difficulty in reusing existing URI tooling.